catalog-info.yaml

```diff
@@ -213,6 +213,62 @@ spec:
 # SECTION END: DRA pipelines
 # ***********************************
 
+# ***********************************
+# SECTION START: Pull requests
+# ***********************************
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: logstash-pull-request-pipeline
+  description: 'Logstash Pull Request pipeline'
+  links:
+    - title: 'Logstash Pull Request pipeline'
+      url: https://buildkite.com/elastic/logstash-pull-request-pipeline
+spec:
+  type: buildkite-pipeline
+  owner: group:logstash
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: logstash-pull-request-pipeline
+      description: ':logstash: Testing for Logstash Pull Requests'
+    spec:
+      pipeline_file: ".buildkite/pull_request_pipeline.yml"
+      branch_configuration: "main 7.* 8.* v7.* v8.* feature/.*"
+      provider_settings:
+        build_pull_request_forks: false
+        build_pull_requests: true # requires filter_enabled and filter_condition settings as below when used with buildkite-pr-bot
+        build_tags: true
+        filter_enabled: true
+        filter_condition: >-
+          build.pull_request.id == null || (build.creator.name == 'elasticmachine' && build.pull_request.id != null)
+      cancel_intermediate_builds: true
+      cancel_intermediate_builds_branch_filter: '!main !7.* !8.*'
+      skip_intermediate_builds: true
+      skip_intermediate_builds_branch_filter: '!main !7.* !8.*'
```

Comment:
Again I am not 100% sure if this is needed for PRs

Reply:
Same as above, we should remove it

Reply:
Addressed in 19e3986beb595dc877fab375b178af172d9f9b3a